ui/templates/vocabulary_create.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block content %}
+    <form action="{% url 'taxonomy.views.edit' %}"
```

Comment:
nit: I don't think it's necessary or desirable to break up the `<form` line. Also, I believe `POST